### DISCUSSION TITLE
import speech_recognition causes an error

### DISCUSSION THREAD

**Discussion by ol-mtvck** [exodrifter/unity-python#38] [Q&A]
Hi, during the "import speech_recognition" error "ImportException: No module named audioop", I did not find such a package, I found only the file "audioop.pyi", threw it in Lib, but it still does not work.

**Comment by exodrifter** [comment]
`.pyi` files are python interface files and won't contain any implementation.

It looks like `audioop` should be a built in package, but the built in packages aren't imported unless you update the search paths. Have you added the `Lib` folder to the search paths of the Python engine?
https://github.com/exodrifter/unity-python/blob/bef6e4e9ddfbbf1eaf7acbbb973e9aa3dd64a20d/Examples/PythonLibrary.cs#L14-L19

**Reply by ol-mtvck** [reply]
I did not change your file, I checked the path to the directory, everything is correct.

**Reply by ol-mtvck** [reply]
void Start()
		{
			var engine = Python.CreateEngine();
			var scope = engine.CreateScope();

			// Add the python library path to the engine. Note that this will
			// not work for builds; you will need to manually place the python
			// library files in a place that your code can find it at runtime.
			var paths = engine.GetSearchPaths();
			paths.Add(Application.dataPath + "/Python/Lib");
			engine.SetSearchPaths (paths);

			string code = @"import speech_recognition";

			var source = engine.CreateScriptSourceFromString(code);
			source.Execute(scope);

			// First, compile the python module to a dll
			// ClrModule.CompileModules();

			// Then, load it
			engine.ImportModule("ctypes");
		}

**Reply by ol-mtvck** [reply]
![error](https://user-images.githubusercontent.com/57463167/145427068-7e80f378-72c9-4445-9ed0-6b932e00856e.png)

**Comment by ol-mtvck** [comment]
Maybe something with the versions?
